src/pages/MonthlyGood/PersonalizedMonthlyGoodLandingPage.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+	<www-page>
+		TESTING
+	</www-page>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+import WwwPage from '@/components/WwwFrame/WwwPage';
+
+const pageQuery = gql`
+	query monthlyGoodPersonalizedLandingPage {
+		contentful {
+			entries(contentType: "page", contentKey: "/lp/monthlygood/personalized")
+		}
+	}
+`;
+
+export default {
+	metaInfo: {
+		title: 'Personalized Monthly Good',
+	},
+	components: {
+		WwwPage,
+	},
+	props: {
+	},
+	data() {
+		return {
+		};
+	},
+	inject: ['apollo', 'cookieStore'],
+	apollo: {
+		query: pageQuery,
+		preFetch(config, client) {
+			return client
+				.query({
+					query: pageQuery,
+				})
+				.then(() => {
+				});
+		},
+		result({ data }) {
+			console.log('data', data);
+		},
+	},
+	mounted() {
+		this.$kvTrackEvent('MonthlyGood', 'EXP-CORE-399-Feb2022', 'b');
```

Comment:
We'll prob want to verify the exp assignment is set before firing this tracking event.

Reply:
I'll add the read query to the created hook 👍